gtsfm/utils/io.py

```diff
@@ -186,7 +236,13 @@ def read_cameras_txt(fpath: str) -> Optional[List[Cal3Bundler]]:
         cam_params = line.split()
         # Note that u0 is px, and v0 is py
         cam_id, model, img_w, img_h, fx, u0, v0 = cam_params[:7]
-        img_w, img_h, fx, u0, v0 = int(img_w), int(img_h), float(fx), float(u0), float(v0)
+        img_w, img_h, fx, u0, v0 = (
```

Comment:
why this change?

Reply:
black formatter

Reply:
remove the parenthesis and retry formatting? does the change persist?

Reply:
Change doesn't persist - artifact of 88 line default setting.